<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_dist_graph_config</class>
 <widget class="QDialog" name="Dialog_dist_graph_config">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>543</width>
    <height>702</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>660</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>521</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>Gráfica Distancia:</string>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>42</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Tamaño:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="tam_graph0">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>40</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="value">
     <double>1.500000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>42</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Color:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>40</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Rojo</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Azul</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Verde</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Gris</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Magenta</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Cyan</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Negro</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Amarillo</string>
     </property>
    </item>
   </widget>
   <widget class="QDoubleSpinBox" name="tam_graph0_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>80</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="value">
     <double>1.500000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>82</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Color:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>82</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Tamaño:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_4">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>80</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <item>
     <property name="text">
      <string>Rojo</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Azul</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Verde</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Gris</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Magenta</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Cyan</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Negro</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Amarillo</string>
     </property>
    </item>
   </widget>
   <widget class="QDoubleSpinBox" name="tam_graph0_3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>120</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="value">
     <double>1.500000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>122</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Color:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>122</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Tamaño:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_5">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>120</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <item>
     <property name="text">
      <string>Rojo</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Azul</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Verde</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Gris</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Magenta</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Cyan</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Negro</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Amarillo</string>
     </property>
    </item>
   </widget>
   <widget class="QDoubleSpinBox" name="tam_graph0_4">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>160</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="value">
     <double>1.500000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>162</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Color:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>162</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Tamaño:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_6">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>160</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>4</number>
    </property>
    <item>
     <property name="text">
      <string>Rojo</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Azul</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Verde</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Gris</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Magenta</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Cyan</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Negro</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Amarillo</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>42</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ID 1:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>82</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ID 2:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>122</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ID 3:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>162</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ID 4:</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>521</width>
     <height>135</height>
    </rect>
   </property>
   <property name="title">
    <string>Eje Y:</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Valor superior:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Valor inferior:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="uppervalue">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>38</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="lowervalue">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>88</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>30</y>
      <width>20</width>
      <height>101</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>64</y>
      <width>161</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Rango Automático</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>510</y>
     <width>521</width>
     <height>134</height>
    </rect>
   </property>
   <property name="title">
    <string>Gráfica Alarma:</string>
   </property>
   <widget class="QDoubleSpinBox" name="tam_graph2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>58</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_3">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>58</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <item>
     <property name="text">
      <string>Rojo</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Azul</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Verde</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Gris</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Magenta</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Cyan</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Negro</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Marrón</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>60</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Color:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Tamaño:</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>370</y>
     <width>521</width>
     <height>135</height>
    </rect>
   </property>
   <property name="title">
    <string>Gráfica Filtro de Media (Modo 1 distancia):</string>
   </property>
   <widget class="QDoubleSpinBox" name="tam_graph1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>62</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Color:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>62</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Tamaño:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>60</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>6</number>
    </property>
    <item>
     <property name="text">
      <string>Rojo</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Azul</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Verde</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Gris</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Magenta</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Cyan</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Negro</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Marrón</string>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog_dist_graph_config</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog_dist_graph_config</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
